apps/src/lib/ui/accounts/ChangeEmailForm.jsx

```diff
@@ -128,22 +128,34 @@ export default class ChangeEmailForm extends React.Component {
             labelDetails={this.emailOptInLabelDetails()}
             error={validationErrors.emailOptIn}
           >
-            <select
-              value={values.emailOptIn}
-              disabled={disabled}
-              tabIndex="1"
-              onKeyDown={this.onKeyDown}
-              onChange={this.onEmailOptInChange}
-              style={{
-                ...styles.input,
-                width: 100
-              }}
-              ref={el => (this.emailOptInSelect = el)}
-            >
-              <option value="" />
-              <option value="yes">{i18n.yes()}</option>
-              <option value="no">{i18n.no()}</option>
-            </select>
+            <>
+              <label>
+                <input
```

Comment:
Since these two inputs are the same (minus `value`), I wonder if there is a way to de-duplicate. Two is such an awkward number . . .

Reply:
ha, I did copy and paste these lines and just replaces "yes" with "no", so I could loop over them. With two, I normally prefer to just list them both but I'm definitely open to push back. It is kind of an awkward number...

Reply:
Yeah! Two is the hardest. Probably makes sense to leave as is?